lib/IRGen/LoadableByAddress.cpp

```diff
@@ -942,11 +942,25 @@ static SILInstruction *createOutlinedCopyCall(SILBuilder &copyBuilder,
   return copy;
 }
 
+// Find the correct debug scope for alloc stack. We want to give to the
+// expanded sequence the correct debug scope so we skip over instructions
+// that aren't lowered to anything real (e.g. debug_value).
+static const SILDebugScope *findAllocStackDebugScope(SILBasicBlock &BB) {
+  auto It = BB.begin();
+  while (It != BB.end()) {
+    if (!isMaintenanceInst(&*It))
```

Comment:
Let's rename isMaintenanceInst to isMetaInstruction. This is the same name we are using in MIR.